README.md

```diff
@@ -6,35 +6,35 @@ This solution starter was created by technologists from IBM.
+- **Denise Knorr** _Operations Director_
+- **Evelyn R. Anderson** _Sr. Security & IP Advisor_
+- **Sebastian Huynh** _Lead Designer_
+- **David Nixon** _Developer_
+- **Gerald Mitchell** _Developer_
+- **Syd Bailey** _Developer_
+- **Ann Umberhocker** _Developer_
+- **Shrey Anand** _Developer_
+- **Alexandra Devine** Designer & IP Strategist
+- **Dylan Zucker** _Data Scientist_
+- **Kristen Ray** _Data Scientist_
+- **Chris Stefano** _Generalist_
+- **Ricky Ellison** _Generalist_
+- **Mike Boone** _Sr. Security & IP Advisor_
 
 # What is the problem?
 
-For centuries, voters in disparate communities, particularly Black voters, have experienced obstacles in casting their votes be it situational or intentionally constructed. Voting has been perceived an intangible and unfair process based on restrictive local processes, changing requirements, regulations, inability to access the correct voting location, and lack of information. As a result millions of minority votes go uncounted, purged, or simply not submitted do to lack of faith and trust in the system.
+American History has shown that voter suppression has led to corruption and a weak democracy that does not reflect the will of the people.Voting has been perceived an intangible and unfair process based on restrictive local processes, changing requirements, regulations, an inability to access the correct voting location, and a lackof information. As a result, millions of minorities, especially Black people, have had their votes go uncounted, purged, or simply not submitted due to lack of faith and trust in the system.
 
-The **Five Fifths Voter** solution starter will provide a capability to help ensure disenfranchised minority voters are Educated, Empowered, and Enabled to overcome setbacks incurred by voter suppresion through achieving the following; clearly identifying the correct polling station, notifying users of all requirements needed for their state and personal circumstances as it pertains to different types of voting, deadlines, and quick access. This solution helps to rewrite the narrative by placing control back in the voters' hands,  generating a voter journey that prioritizes their individual lifestyle and needs.
+The **Five Fifths Voter** solution starter focuses on addressing thekey areas of voter suppression such as voter registration, voter ID laws, voter registration restrictions, voter purging, felony disenfranchisement, and gerrymandering all of which are contributors to disenfranchising minorities and people with disabilities.The solutionhelps with these challenges by providing a capability to help ensure disenfranchised minority voters are educated, empowered, and enabled to overcome setbacks incurred by voter suppression. Five Fifths Voter helps to identify the correct polling stations, notifies users of requirements needed for their state, and personal circumstances as it pertains to different types of voting, deadlines, and quick accessto information.
```

Comment:
thekey -> the key
solutionhelps -> solution  helps  -> (suggestion replace "solution helps with" to be more dynamic)  confronts | counteracts | eases | surmounts... )
solutionhelps ... capability to help ensure  -> too many "helps" see suggestion above.
accessto -> access to